src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2028,6 +2028,21 @@ Here is the ordered list of rules for the EDF policy:
 
 6. job with earliest submission time.
 
+==== Email notificaions
+
+EDF policy sends 3 types of email notifications:
+
+1. [UNDER CONSTRUCTION] when job deadline is at risk (if we do not start job in next X minutes we will miss its deadline)
```

Comment:
- When (Capital W). But I would prefer If rather than when, because it's more like a conditional sentence.
- The [Under Construction] tag is a bit misleading because at first glance it looks like this section of the documentation is under construction and not the feature itself.
Use something like (Arriving soon) between parentheses instead for a literature-like style.